rotkehlchen/oracles/structures.py

```diff
@@ -9,7 +9,6 @@ class CurrentPriceOracle(OracleSource):
     CRYPTOCOMPARE = auto()
     UNISWAPV2 = auto()
     UNISWAPV3 = auto()
-    SADDLE = auto()
     MANUALCURRENT = auto()
```

Comment:
Hmm ... I did not check but are we **absolutely 100% sure** that nothing depends on the order of these enums and that the value is just used in runtime only. They are not saved in the DB ad numerical values right?

Cause you are removing one and moving all the ones after it down by `-1` here

Reply:
> Yes, I checked and is not needed. But this enum is the reason to remove saddle also in the old db upgrades. The reason is that the settings call deserialize on this class and that can lead to deserialization error if saddle is not removed or handled

That just means the old upgrade is built wrong, no? It should not rely on functions that can change